circuits/voteverifier/inputs.go

```diff
@@ -0,0 +1,176 @@
+package voteverifier
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"math"
+	"math/big"
+
+	"github.com/consensys/gnark-crypto/ecc"
+	"github.com/consensys/gnark-crypto/ecc/bls12-377/fr/mimc"
+	"github.com/consensys/gnark/frontend"
+	"github.com/consensys/gnark/std/algebra/emulated/sw_bn254"
+	"github.com/consensys/gnark/std/math/emulated"
+	gnarkecdsa "github.com/consensys/gnark/std/signature/ecdsa"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/vocdoni/arbo"
+	primitivestest "github.com/vocdoni/gnark-crypto-primitives/testutil"
+	"github.com/vocdoni/vocdoni-z-sandbox/circuits/ballotproof"
+	"go.vocdoni.io/dvote/util"
+)
+
+// VoteVerifierTestResults struct includes relevant data after VerifyVoteCircuit
+// inputs generation
+type VoteVerifierTestResults struct {
```

Comment:
Do we need to export these test types? Please review all of them.

Reply:
they make the tests more readable and easy to follow, and they are used in multiple tests